runway/hooks/staticsite/auth_at_edge/templates/shared.py

```diff
@@ -0,0 +1,351 @@
+# pylint: skip-file
```

Comment:
Same as above, skip-file should be dropped here

Reply:
Still needed. It picks up on the redirect return values. Large discussion with Kyle about this in an earlier conversation. Leaving in-tact.